gammapy/utils/scripts.py

```diff
@@ -91,6 +91,26 @@ def make_path(path):
     # Otherwise this can result in cryptic errors later on
     return Path(expandvars(str(path)))
 
+def name_from_path(filepath):
```

Comment:
What does `name_from_path` do? It's never called?

This shouldn't be needed, we use `pathlib.Path` and that has a lot of convenience methods.

Please delete.
Or show a caller & test & use case, and we can try to make it simpler.